.gitignore

```diff
@@ -5,7 +5,6 @@ node_modules
 npm-debug.log
 .node_history
 package-lock.json
-dist
```

Comment:
is the inclusion of `dist` and all the files there intended? what are the steps then to upload the code, since those files are generated by tools?

Reply:
for some reason npm is not publishing the `dist` folder, that is why we had compiling issues and the bundle does not represent our transpiled code! This change is temporary until we fix npm @developit 😂